trestle/core/resolver/modify.py

```diff
@@ -361,6 +365,11 @@ def _modify_controls(self, catalog: cat.Catalog) -> cat.Catalog:
                     self._set_parameter_in_control_or_loose(set_param)
             alters = self._profile.modify.alters
 
+        # an add with no by-id applies to the control
+        # if position is starting it should appear immediately after title
+        # if position is ending it should appear at end
+        # if not specified it defaults to ending
+        # if no by-id then before is treated as starting and after is treated as ending
```

Comment:
Shall this be formalised in the function docs?

Reply:
this is a private function so I don't think it warrants full documentation, and the point of those comments is just a reminder for the logic of how position works, and how the defaults are specified by NIST.

Reply:
The problem is that comments like that became outdated as code develops, easier to change docs rather than comment. But fine to keep for now